namespaces/live-1.cloud-platform.service.justice.gov.uk/calculate-release-dates-api-dev/06-certificate.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: calculate-release-dates-api-dev.hmpps.service.justice.gov.uk
+  namespace: calculate-release-dates-api-dev
+spec:
+  secretName: calculate-release-dates-api-dev-cert
+  issuerRef:
```

Comment:
This secret name doesn't need the `-dev` - it will be called the same name in all envs - so no dev specific here.

Reply:
I think this has been addressed